thewalrus/quantum/gaussian_state_tests.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2019-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Tests for various properties of covariance matrices as well as fidelity
+calculations for Gaussian states.
```

Comment:
I would avoid using 'tests' here and in the module name, just to avoid any confusion between unittests. Instead, maybe `gaussian_checks.py`?

Reply:
Agreed! `gaussian_checks` is better.